src/engines/kotor/module.cpp

```diff
@@ -22,6 +22,8 @@
  *  The context needed to run a Star Wars: Knights of the Old Republic module.
  */
 
+#include <src/engines/kotor/gui/loadscreen/loadscreen.h>
```

Comment:
Please use `#include ""` instead of `#include <>` for our own includes. This one should also be way down in the list, and of course below the Boost.Bind one.